db/migrate/20181013195028_add_confidential_to_applications.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
```

Comment:
do we really need this here?

Reply:
This migration was generated by the doorkeeper upgrade procedure, from [this template](https://github.com/doorkeeper-gem/doorkeeper/blob/3b61b6c3e99a309c8e5e25f86ee6ac2c8131b05d/lib/generators/doorkeeper/templates/add_confidential_to_applications.rb.erb). I agree that we'd better use our own style here.